File: OnlineEyeClinic Back end/src/main/java/com/cg/onlineeyeclinic/services/LoginService.java
package com.cg.onlineeyeclinic.services;

//******************************Login Service Interface***************************//

public interface LoginService 
{
	//admin login
	boolean adminLogin(String username,String password);
	
	//patient login
	boolean patientLogin(String username,String password);
	
	//doctor login
	boolean doctorLogin(String username,String password);
}

File: online-eye-clinic/src/app/services/login.service.ts
import { HttpClient } from '@angular/common/http';
import { Injectable } from '@angular/core';
import { Observable } from 'rxjs';

@Injectable({
  providedIn: 'root'
})
export class LoginService {

  constructor(private httpClient: HttpClient) { }

  private baseURL = "http://localhost:9090/onlineeyeclinic/login";

  getAdminLogin(username:string,password:string):Observable<Boolean>
  {
    return this.httpClient.get<Boolean>(`${this.baseURL}/admin/${username}/${password}`);
  }

  getDoctorLogin(username:string,password:string):Observable<Boolean>
  {
    return this.httpClient.get<Boolean>(`${this.baseURL}/doctor/${username}/${password}`);
  }

  getPatientLogin(username:string,password:string):Observable<Boolean>
  {
    return this.httpClient.get<Boolean>(`${this.baseURL}/patient/${username}/${password}`);
  }




}
